ci/jenkins/test.sh

```diff
@@ -1002,15 +1088,14 @@ fi
 
 trap clean_antrea EXIT
 if [[ ${TESTCASE} =~ "windows" ]]; then
-    if [[ ${TESTCASE} =~ "containerd" ]]; then
-        WINDOWS_YAML_SUFFIX="windows-containerd-with-ovs"
-        build_and_deliver_antrea_windows_and_linux_containerd_images
-        if [[ ${TESTCASE} =~ "e2e" ]]; then
-            run_e2e_windows
-        else
-            run_conformance_windows_containerd
-        fi
+    WINDOWS_YAML_SUFFIX="windows-containerd-with-ovs"
```

Comment:
Why hardcode the yaml name here? In userspace e2e test we should use `antrea-windows-containerd.yml` because ovs is not started in container. `antrea-windows-containerd-with-ovs.yml` is used in windows e2e, conformance and networkpolicy tests.